File: codes/src-tgt/cpp-java/HumanEval_84_solve.java
### C++ version

#include<assert.h>
#include<bits/stdc++.h>
#include<vector>
#include<sstream>
#include<regex>
#include<map>
#include<set>
std::string solve(long N) {
    std::string s = std::to_string(N);
    long sum = 0;
    for (int i = 0; i < s.size(); i++) {
        sum += s[i] - '0';
    }
    std::bitset<64> b(sum);
    return b.to_string().substr(b.to_string().find('1'));
}


### Java version

import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    public static String solve(long N) {
        return Long.toBinaryString(Long.toString(N).chars().map(c -> c - '0').sum());
    }
    

}

File: test/before_train/trainset_and_testset/java-cpp/HumanEval_84_solve.cpp
#include<assert.h>
#include<bits/stdc++.h>
#include<vector>
#include<sstream>
#include<regex>
#include<map>
#include<set>
#include<unordered_map>
#include<unordered_set>
std::string solve(long N) {
    std::stringstream ss;
    ss << N;
    return ss.str();
}
int main() {
    auto candidate = solve;
    assert(candidate((1000)) == ("1"));
    assert(candidate((150)) == ("110"));
    assert(candidate((147)) == ("1100"));
    assert(candidate((333)) == ("1001"));
    assert(candidate((963)) == ("10010"));
}
